src/frontend/components/LiveVideoPanel/index.tsx

```diff
@@ -98,7 +99,12 @@ export const LiveVideoPanel = ({ video }: LiveVideoPanelProps) => {
   }
 
   return (
-    <Box background="white" flex direction="column">
+    <Box
+      background="white"
+      flex
+      direction="column"
+      style={{ minWidth: '210px' }}
+    >
```

Comment:
```suggestion
    <Box
      background="white"
      flex
      direction="column"
      width={{ min: '210px' }}
    >
```

but I think panel width is computed in js and min width should not be handle in css